deploy/infra/aws/terraform/cloud-native/cluster-infra/output.tf

```diff
@@ -35,15 +35,35 @@ output "cluster_cp_worker_node_execution_role" {
 
 output "cp_ssh_rsa_key_pair" {
   description = "RSA key pair to use during Cloud-Pipeline deployment"
-  value       =  module.ssh_rsa_key_pair
+  value       = module.ssh_rsa_key_pair
 }
 
 output "cp_etc_bucket" {
   description = "Cloud-pipeline etc bucket name"
-  value = module.s3_etc.s3_bucket_id
+  value       = module.s3_etc.s3_bucket_id
 }
 
 output "cp_docker_bucket" {
   description = "Cloud-pipeline docker registry bucket name"
-  value = module.s3_docker.s3_bucket_id
+  value       = module.s3_docker.s3_bucket_id
+}
+
+output "root_pass_secret" {
```

Comment:
rename this output to `rds_root_pass_secret` and describe it better in the description